packages/doctor/lib/factory.js

```diff
@@ -19,3 +22,12 @@ let newDoctor = (opts) => {
 };
 
 export default newDoctor;
+
+/**
+ * @typedef CheckList
+ * @property {import('./doctor').DoctorCheck[]} generalChecks
```

Comment:
When you `import` the same type more than once, create a new `@typedef`; all the `import`s become too noisy otherwise.